substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2631,6 +2674,20 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 			Self::do_claim_commission(who, pool_id)
 		}
+
+		/// Top up the deficit or withdraw the excess ED from the pool.
+		///
+		/// When a pool is created, the pool depositor transfers ED to the reward account of the
+		/// pool. ED is subject to change and over time, the deposit in the reward account may be
+		/// insufficient to cover the ED deficit of the pool or vice-versa where there is excess
+		/// deposit to the pool. This call allows the pool operator to adjust the ED deposit of the
+		/// pool.
+		#[pallet::call_index(21)]
+		#[pallet::weight(T::WeightInfo::adjust_ed_deposit())]
+		pub fn adjust_ed_deposit(origin: OriginFor<T>, pool_id: PoolId) -> DispatchResult {
```

Comment:
nit: how about calling this extrinsic `adjust_pool_deposit` instead?

Reply:
Yeah I like `adjust_pool_deposit` better

Reply:
Could deposit be misunderstood as related to `MinCreateBond`?